mitodl/ocw-hugo-themes
# Make notifications accessible

# What are the relevant tickets?
Closes #1218 

# Description (What does it do?)
Makes notifications accessible by adding `aria-label` and `role`.

# How can this be tested?
1. Checkout to this branch
3. Go to your local ocw-content-rc directory, and open up `ocw-www/content/2023-24hc.md`. Change `draft` to `false`. This should enable the notification.
4. `yarn start www`
5. Go to `localhost:3000` and you should see a notification bar on top of the page. You can also access it at: http://localhost:3000/notifications/2023-24hc/ (ignore the duplicated text at bottom)
6. The changes in this PR are with respect to the Close button on the right. It should now say "Close Notification, Button" if you run it by a screen reader.
7.  Make sure the close notification button works.
